CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 - feat: Add support for `bun@1.2`'s plaintext `bun.lock` file
 - fix: only skip tsconfig on pnpm workspace project
 - fix: update stylelint order and use spaces instead of tabs
+- **BREAKING** feat: Raise `pkg.engines.node` to `^20.15.0`
+- **BREAKING** feat: Remove `eslint-plugin-prettier`
+  - treat linting and formatting as separate concerns
+  - this obviates the `_guiltily_disable_prettier` option
+- feat: Change prettier's `trailingComma` setting to `'all'` (default)
+- feat: Add `skipLibCheck` to default `tsconfig.json`
```

Comment:
Nit:
Sort items: Breaking>feat>fix>docs for readability.
(This can happen after merge.)

Reply:
Yea, the only breaking change is the min node version change, but I would release this as v9.
Experimented with the flatconfig update again a couple of months ago and didn't get it fully working.